ENIGMAsystem/SHELL/Bridges/xlib-OpenGL1/attribs.cpp

```diff
@@ -0,0 +1,13 @@
+#include <GL/gl.h>
+#include <GL/glx.h>
+
+namespace enigma {
+
+int context_attribs[] = {
+    GLX_CONTEXT_MAJOR_VERSION_ARB, 1,
+    GLX_CONTEXT_MINOR_VERSION_ARB, 1,
+    GLX_CONTEXT_PROFILE_MASK_ARB, GLX_CONTEXT_COMPATIBILITY_PROFILE_BIT_ARB,
+    None
```

Comment:
This should break everything, even though you are requesting a compatibility profile, this makes it always be version 1.1 so surfaces will never work. Try a game with surfaces to confirm.

Reply:
I dont have any games other than catch clown. And i just guessed at ogl 1 version. What version you want there?